service-search-nls/src/main/java/fi/nls/oskari/search/util/ELFGeoLocatorCountries.java

```diff
@@ -31,53 +30,118 @@
     private long lastSync;
     private String serviceURL;
     private ELFGeoLocatorSearchChannel channel;
-    private Map<String, String> countries = new HashMap<>();
+    private static Map<String, String> countryMap = new HashMap<>();
```

Comment:
This shouldn't be static

Reply:
Is it not good that it refers to the same map in all instances.